### PR TITLE
Modularizes Gympie sprites, properly this time, adds gympies back, & tweaks aspects.

### PR DESCRIPTION
**Pull request by DrDuckedGoose**
## About The Pull Request
All gympie related sprites live in their own folder now, this will later be populated by more planned additions by the author. Additionally adds gympie content back, adjusts gympie-growing sprites, and change gympie-awakening chem to l4z.

## Why It's Good For The Game
Adds them back after Terra prioritized fairy grass.

## Changelog
:cl:
Returns gympie-gympie seeds & mob.
Adds separate file for gympie & future sprites.
Adjusts gympie sprites and awakening chem. 
/:cl:

### CODE REVIEW

**Comment by MisterMecky** [comment]
While this may not cause any merge conflicts in the future, you'll still want to move all your files to be inside the 'austation' folder. 

`austation/code/modules/uplink/uplink_items.dm` Is in the correct location

`code/modules/hydroponics/grown/gympie_gympie.dm`
 move this to
`austation/code/modules/hydroponics/grown/gympie_gympie.dm`

`code/modules/mob/living/simple_animal/hostile/gympie_gympie.dm`
move this to
`austation/code/modules/mob/living/simple_animal/hostile/gympie_gympie.dm`

` icons/obj/hydroponics/popes_selection.dmi`
move this to
`austation/icons/obj/hydroponics/popes_selection.dmi`
(Don't forget to change the icons directory location inside the code to reflect this change)

Then you can append includes to the the austation.dme
`#include "austation/code/modules/hydroponics/grown/gympie_gympie.dm"`
`#include "austation/code/modules/mob/living/simple_animal/hostile/gympie_gympie.dm"`

File: austation.dme [hunk 1]
```diff
@@ -297,4 +297,7 @@
 #include "austation\code\modules\events\disease_outbreak.dm"
 #include "austation\code\modules\events\electrical_storm.dm"
 #include "austation\code\modules\events\space_dragon.dm"
+#include "austation/code/modules/hydroponics/grown/gympie_gympie.dm"
+#include "austation/code/modules/mob/living/simple_animal/hostile/gympie_gympie.dm"
+#include "austation/icons/obj/hydroponics/popes_selection.dmi"
```
Comment (MisterMecky) [inline austation.dme]: I don't think you need to include icon files

File: austation.dme [hunk 2]
```diff
@@ -297,4 +297,7 @@
 #include "austation\code\modules\events\disease_outbreak.dm"
 #include "austation\code\modules\events\electrical_storm.dm"
 #include "austation\code\modules\events\space_dragon.dm"
+#include "austation/code/modules/hydroponics/grown/gympie_gympie.dm"
```
Comment (MisterMecky) [inline austation.dme]:
```suggestion
#include "austation\code\modules\hydroponics\grown\gympie_gympie.dm"
```

File: austation.dme [hunk 3]
```diff
@@ -297,4 +297,7 @@
...
+#include "austation/code/modules/mob/living/simple_animal/hostile/gympie_gympie.dm"
```
Comment (MisterMecky) [inline austation.dme]:
```suggestion
#include "austation\code\modules\mob/living\simple_animal\hostile\gympie_gympie.dm"
```

**Comment by DrDuckedGoose** [comment]
I clearly misunderstood what you were asking. Job's done boss.

**Comment by MisterMecky** [comment]
Can you change those forward slashes ( / ) to backslashes ( \ ) in the includes

**Review by MisterMecky — APPROVED** [review]

**Pull request merged**